### PR TITLE
fix: added warning when the unity inputsystem is not used

### PR DESCRIPTION
**Pull request by SimonTheSourcerer**

### CODE REVIEW

**Review by Gusinuhe — DISMISSED** [review]

File: Editor/InputSystemChecker.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Innoactive.Creator.Core.Input;
+using UnityEditor;
+
+[InitializeOnLoad]
+public static class InputSystemChecker
+{
+    private const string message =
+        "From Creator version 2.11 onwards Unity's new Input System is required but not added to your project yet." +
```
Comment (burnedikt) [inline Editor/InputSystemChecker.cs:9]:
```suggestion
        "From Creator version 2.11 onwards Unity's new Input System is required." +
```

If we say it is not added, it just raises the question why not ;-)

File: Editor/InputSystemChecker.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        try
+        {
+            // This will throw an InvalidOperationException when no concrete implementation is found.
+            Type type = InputController.ConcreteType;
+        }
+        catch (InvalidOperationException)
+        {
+            if (CreatorProjectSettings.Load().IsFirstTimeStarted == false)
+            {
+                EditorUtility.DisplayDialog("Attention required!", message, "Understood");
```
Comment (burnedikt) [inline Editor/InputSystemChecker.cs:27]: Once I've dismissed this warning, I will never see it again until next Editor launch? Or will it reoccur after every compile?

Comment (SimonTheSourcerer) [reply Editor/InputSystemChecker.cs:27]: EVERY compile

Comment (SimonTheSourcerer) [reply Editor/InputSystemChecker.cs:27]: As the whole creator will break on play, this is intended to be annoying.

**Review by tomwim — APPROVED** [review]

**Pull request merged**